src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinBuilderJvmJdepsTest.kt

```diff
@@ -46,6 +46,7 @@ class KotlinBuilderJvmJdepsTest(private val enableK2Compiler: Boolean) {
   val ctx = KotlinJvmTestBuilder()
 
   val TEST_FIXTURES_DEP = Dep.fromLabel(":JdepsParserTestFixtures")
+  val TEST_FIXTURES2_DEP = Dep.fromLabel(":JdepsParserTestFixtures2")
```

Comment:
To enable test cases where Java deps depend on other Java deps I added a second set of java test fixtures here